content/rs/administering/monitoring-metrics/prometheus-integration.md

```diff
@@ -0,0 +1,324 @@
+---
+Title: Redis Enterprise Software Integration with Prometheus
+description: 
+weight: $weight
+alwaysopen: false
+---
+Redis Enterprise lets you connect your Prometheus or Grafana server to your Redis Enterprise cluster in order to collect and display metrics data. Metrics are exposed at the node, database, shard and proxy levels.
```

Comment:
@bmansheim @Paz-Yanover Grafana doesn't connect to Redis Enterprise, its querying Prometheus.